src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,71 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+
+@Entity
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne
+    private Movie movie;
+    @OneToOne
+    private CinemaHall cinemaHall;
```

Comment:
Are you sure that the session cannot be held in several hall or movie cannot be in several sessions